### ISSUE TITLE
Web - Allow allocation to be changed when editing data record

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#28]
Once the data move feature is implemented, users should also be able to relocate individual records to different allocations/repos via the data edit dialog. If the allocation is changed a background data transfer would be initiated. Need to warn users the same as using the relocate feature.

**Comment by JoshuaSBrown** [comment]
What is the relocate feature? Is the "relocate feature" already implemented? Is the "relocate feature" the same as dragging a record from one allocation to another?

**Comment by JoshuaSBrown** [comment]
@dvstans ping.

**Comment by dvstans** [comment]
Yes, it's the same as dragging to another allocation. This isn't hard to implement, just something that needs to be done.
